Platform.PCL/Realm.Sync.PCL/UserPCL.cs

```diff
@@ -28,6 +28,46 @@ namespace Realms.Sync
     /// </summary>
     public class User : IEquatable<User>
     {
+        #region static
+
+        /// <summary>
+        /// Gets the currently logged-in user. If none exists, null is returned. If more than one user is currently logged in, an exception is thrown.
```

Comment:
The SyncUser has 3 states - Active, LoggedOut, and Error. Currently logged-in users are those whose state is Active.